data/json/mapgen/private_resort.json

```diff
@@ -0,0 +1,1841 @@
+        { "chance": 50, "item": "restaur_table", "x": [ 18, 20 ], "y": [ 15, 16 ], "repeat": [ 3, 9 ] },
+        { "chance": 50, "item": "restaur_table", "x": [ 5, 6 ], "y": [ 20, 20 ], "repeat": [ 2, 5 ] },
+        { "chance": 50, "item": "restaur_table", "x": [ 10, 11 ], "y": [ 19, 20 ], "repeat": [ 2, 8 ] },
+        { "chance": 50, "item": "restaur_table", "x": [ 18, 20 ], "y": [ 19, 20 ], "repeat": [ 3, 9 ] },
+        { "chance": 50, "item": "restaur_fridge", "x": [ 23, 23 ], "y": [ 1, 6 ], "repeat": [ 3, 15 ] }
+      ],
+      "place_monsters": [
+        { "monster": "GROUP_RESORT_MIXED", "x": [ 5, 23 ], "y": [ 0, 23 ], "density": 0.75 },
+        { "monster": "GROUP_RESORT_MIXED_STAFF", "x": [ 7, 23 ], "y": [ 0, 6 ], "density": 0.2 }
+      ]
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": [ "p_resort_1sw" ],
+    "weight": 1000,
+    "object": {
+      "fill_ter": "t_floor",
+      "rows": [
+        ".F..W  WTSTW^^W^^WTSTW  ",
+        ".F..W  W   W $W$ W   W  ",
+        ".F..W  +   +  W  +   +  ",
+        ".F..W  WWWWWWWWWWWWWWW  ",
+        ".F..W  +   +  W  +   +  ",
+        ".F..W  W   W $W$ W   W  ",
+        ".F..W  WTSTW^^W^^WTSTW  ",
+        ".F..WWWWWWWWWWWWWWWWWWWW",
+        ".F......................",
+        ".F.____________________x",
+        ".F.____________________x",
+        ".F.____________________x",
+        ".F.:::::::______:::::::x",
+        ".F.:::::::______:::::::x",
+        ".F.____________________x",
+        ".F.____________________x",
+        ".F.____________________x",
+        ".F.:::::::______:::::::x",
+        ".F.:::::::______:::::::x",
+        ".F.____________________x",
+        ".F.____________________x",
+        ".F.____________________x",
+        ".F.:::::::______:::::::x",
+        ".F.:::::::______:::::::x"
+      ],
+      "palettes": [ "p_resort_palette_main_floor" ],
+      "place_monsters": [ { "monster": "GROUP_RESORT_MIXED", "x": [ 5, 23 ], "y": [ 0, 7 ], "repeat": [ 0, 1 ], "density": 0.2 } ],
+      "toilets": { "$": {  } },
+      "place_items": [
+        { "chance": 50, "item": "restaur_bath", "x": [ 12, 13 ], "y": [ 0, 2 ], "repeat": [ 3, 6 ] },
+        { "chance": 50, "item": "restaur_bath", "x": [ 15, 16 ], "y": [ 0, 2 ], "repeat": [ 3, 6 ] },
+        { "chance": 50, "item": "restaur_bath", "x": [ 12, 13 ], "y": [ 4, 6 ], "repeat": [ 3, 6 ] },
+        { "chance": 50, "item": "restaur_bath", "x": [ 15, 16 ], "y": [ 4, 6 ], "repeat": [ 3, 6 ] },
+        { "chance": 25, "item": "stash_drugs", "x": [ 12, 13 ], "y": [ 0, 2 ], "repeat": [ 1, 2 ] },
+        { "chance": 25, "item": "stash_drugs", "x": [ 15, 16 ], "y": [ 0, 2 ], "repeat": [ 1, 2 ] },
+        { "chance": 25, "item": "stash_drugs", "x": [ 12, 13 ], "y": [ 4, 6 ], "repeat": [ 1, 2 ] },
+        { "chance": 25, "item": "stash_drugs", "x": [ 15, 16 ], "y": [ 4, 6 ], "repeat": [ 1, 2 ] }
+      ],
+      "place_vehicles": [
```

Comment:
This seems like a big stock of pristine cars

Reply:
Unless I am wrong, status -1 is slightly damaged, 1 is heavily damaged and status 0 is pristine.

Reply:
Ah, my mistake there possibly. You still might want to change the chance of spawning a little. I assume people would have tried to flee the place when zombies started showing up

Reply:
I will tweak them down to 20, I suppose that's reasonable to have 2-3 cars still there.